edge/discussion_materials/ONNX Edge scope and profile definition.md

```diff
@@ -0,0 +1,72 @@
+
+## ONNX Edge profile
+
+The ONNX Edge profile is described with the following:
+
+1. Attributes / characteristics: Power, Size, accuracy, speed, Security, etc.
+
+2. ONNX operations subset
+
+3. Other ONNX related limitations 
+
+Work to define strict subsets of the operator sets which apply to edge devices which avoid complex operations imposed by limitations on edge devices with high computational complexity. Exclude use-cases not applicable to the edge (e.g. large scale training scenarios)
+
+
+### Edge profile attributes
+
+#### Accuracy
+
+The ONNX model zoo collected many wide-used models for image classification, face recognition and image segmentation. While the accuracy number varies, the top-5 accuracy seems acceptable with VGG, resent and mobile net are all above or close to 90%. Accuracy is critical for scenarios like smart city, face recognition and smart monitoring. If put these models in edge device in actual use, primary work is to keep the accuracy still in a decent level while compressing the model and changing ops to low-bit version. Detailed work can be achieved cooperating with the quantization WG.
+
+#### Size
+
+Size here refers to both the model on-disk storage size and the memory that the runtime reqiures to run the inference. In ONNX 1.4 release, support for large models (larger than 2GB) and store the data externally is added as a new feature. But in edge device, we do not want such huge models running and consuming the precious memory. There will always be compromises between performance, speed, cost and consumption. There are many ways to compress our models while keeping the accuracy like changing to quantized ops, model compression etc. 
+
+#### Latency
+
+In many scenarios, fast processing speed and low latency are required. Vendor-specific runtime optimization are encouraged while we should also keep our optimization work going. works on the speed over laps with some of the work on the bandwidth, since a broader bandwidth always provide less latency thus improving overall speed.
+
+#### Power consumption
+
+Power consumption is critical for an edge device. Methods are to be defined to reduce the power consumption/extend the service time of this edge device while maintaining the same level of performance:
+
+1. A larger battery which extend the servie time of an edge device.
+
+2. A better chipset which utilized with more advanced algorithms and technique to lower consumption in every clock time.
+
+3. New scheduling system to switch off some unused applicatio or parts
+
+4. Better cooling system that doesn't consume like the old fashioned fan
+
+5. etc. 
+
+
+#### Data locality
+
```

Comment:
To set the context for paragraphs that follow below, I would start by saying that data locality is tightly connected with data privacy and security aspects of a scenario.